library/alignment.h

```diff
@@ -155,13 +155,13 @@ inline void mbedtls_put_unaligned_uint64(void *p, uint64_t x)
  * Detect Clang built-in byteswap routines
  */
 #if defined(__clang__) && defined(__has_builtin)
```

Comment:
Ah, we should actually check for clang first, then (in an else) for gcc, since clang also defines `__GNUC__`